shard/src/ext/latex_cog.py

```diff
@@ -40,7 +40,7 @@ def __init__(self, bot):
     @cooldown(2, 15, BucketType.user)
     async def latex(self, ctx, *latex):
         """
-        Use: !latex [valid latex code]
+        Usage: !latex [valid latex code]
```

Comment:
this line is automatically added to the help output based on the args. the first line should be a short help version